engines/freescape/unpack.cpp

```diff
@@ -109,6 +109,7 @@ void unpack_data(unsigned char *unpacked_data, unsigned char *buf, unsigned int
 	unsigned char *save_buf = NULL;
 	unsigned char *save_unp = NULL;
 	unsigned int cur_unpacked_data_size = 0x00;
+	unsigned int data_left = 0x00;
```

Comment:
The variable name seems incorrect.
This represents the data consumed: save_buf represents the start of the buffer and buf the current position.
Subtracting save_buf to buf equals to the current index of data (or the quantity consumed).

Reply:
The codes from here: https://github.com/w4kfu/unEXEPACK/blob/master/unpack.c I'm not sure if we should change this variable name.